### DISCUSSION TITLE
Cache revalidate sends double requests

### DISCUSSION THREAD

**Discussion by stajics** [vercel/next.js#44944] [Help]
I am using a simple server component data fetching and revalidation in my page. I am fetching the data from a 3rd party api
```
export const revalidate = 60;

async function getSomeData() {
  const res = await axios(
    "some 3rd paty api"
  );
  return res.data;
}
```
I noticed the number of requests made to the 3rd party API service increase by 2 when the page revalidates. I am curious why is that?

**Comment by havran** [comment]
Probably related with my issue https://github.com/vercel/next.js/issues/44966
